### ISSUE TITLE
Remove returns from anonymous `addEventListener` handlers

### ISSUE THREAD

**Issue by GoodForOneFare** [Shopify/javascript#72]
Vanilla and jQuery handlers treat return values very differently.  Removing (completely meaningless) explicit return values from `addEventListener` handlers should discourage conflating of techniques.

Example:
```
document.addEventListener('input', event => {
  if (event.target.type === 'hidden') {
    event.preventDefault();
    return event.stopPropagation();
  }
}, true);
```

**Comment by fandy** [comment]
https://github.com/Shopify/javascript/pull/76

**Comment by fandy** [comment]
Fixed in #76
